### PR TITLE
Arnej/split tensor eval expensive

### PR DESCRIPTION
**Pull request by arnej27959**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

NOTE: this just splits the ruby / graphs etc, it's still same .sd and same feed, will look into that in a follow-up PR
@geirst please review
@hmusum / @aressem please review the "noise reduction" commits - maybe the "ps" output is useful to you?

### CODE REVIEW

**Comment by aressem** [comment]
@arnej27959 Looks good 👍  The ps output was used in some debugging a while ago, but forgot to comment out / remove again.

File: tests/performance/tensor_eval/tensor_eval_match.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+require 'performance/tensor_eval/tensor_eval'
+
+class TensorEvalExpensivePerfTest < TensorEvalPerfTest
```
Comment (geirst) [inline tests/performance/tensor_eval/tensor_eval_match.rb:5]: Please rename to a unique name as it shares name with tensor_eval_matrix.rb.

File: tests/performance/tensor_eval/tensor_eval_matrix.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+require 'performance/tensor_eval/tensor_eval'
+
+class TensorEvalExpensivePerfTest < TensorEvalPerfTest
```
Comment (geirst) [inline tests/performance/tensor_eval/tensor_eval_matrix.rb:5]: Please rename to a unique name as it shares name with tensor_eval_match.rb.

**Review by geirst — CHANGES_REQUESTED** [review]
Two classes must be renamed, otherwise LGTM.

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**